platform-includes/configuration/drain-example/apple.mdx

```diff
@@ -1 +1,10 @@
 The Apple SDK automatically stores the Sentry events on the device's disk before shutdown.
+
+There are times where you may need to manually call `close()` to ensure that events are sent before the app terminates. For example, if an app user revokes some privacy consent and the app can't use Sentry anymore, calling `close()` will still send the events to Sentry.
+
+```swift
+SentrySDK.close()
+```
+
+After closing, the client cannot be used anymore. It's important to only call
+`close()` immediately before shutting down the application.
```

Comment:
I think we don't need that. It's confusing. We don't need to call close before shutting down the application, and on Cocoa almost nobody interacts with the client. Everybody interacts with the static API such as `SentrySDK.capture`
```suggestion
```